README.md

```diff
@@ -91,6 +91,8 @@ The license properties (rules) are stored as a bulleted list within the licenses
 * `disclose-source` - Source code must be made available when distributing the software.
 * `network-use-disclose` - Users who interact with the software via network are given the right to receive a copy of the corresponding source code.
 * `same-license` - Modifications must be released under the same license when distributing the software. In some cases a similar or related license may be used.
+* `same-license--file` - Modifications of existing files must be released under the same license when distributing the software. In some cases a similar or related license may be used.
```

Comment:
@benbalter to mark as refinement without having to loop through rules to check for substring, or having another field indicating refinement, or having knowledge of specific rules in software using the rules. I use this in the super-ugly appendix code. If `--` is itself super-ugly I can switch to `-` and use slightly different logic.